src/icons/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export const encodeSVG = (svg: string) => encodeURI(svg).replace(/#/g, '%23');
+
+export type IconProps = {
+  alt?: string;
+  fill?: string;
+  height?: string;
+  style?: object;
+  width?: string;
+};
+
+export type BaseSvgProps = {
+  svg: string;
+  alt?: string;
+  fill?: string;
+  height?: string;
+  size?: string;
+  style?: object;
+  width?: string;
+};
+
+export const BaseSvg = ({
+  svg,
+  size = '1em',
+  height = size,
+  width = size,
+  style,
+  alt = '',
+  ...props
```

Comment:
I know that ...props is legitimate, here. But, I think we should not use and a needed prop when needed